src/QsCompiler/Core/SymbolTable/NamespaceManager.fs

```diff
@@ -76,8 +76,8 @@ type NamespaceManager
     /// A namespace with the given name was not found, or the source file does not contain the namespace.
     /// </exception>
     let openNamespaces (nsName, source) =
-        let isKnownAndNotAliased (kv: KeyValuePair<_, _>) =
-            if not (isNull kv.Value) then
+        let isKnownAndNotAliased (kv: KeyValuePair<string, ImmutableHashSet<string>>) =
+            if not (kv.Value.Contains("")) then
```

Comment:
Pretty sure this is wrong. Even a namespace has an alias, we shouldn't skip it when looking for unqualified symbols

Reply:
Oops, I misread, this is legit